code/modules/codex/entries/faction_codex.dm

```diff
@@ -0,0 +1,88 @@
+/datum/codex_entry/TG_faction
+    display_name = "Terran Government (faction)"
+    lore_text = "The Terran Government, known informally as the TG, is a militaristic type of democracy, requiring military service and bolstering a \
+    a large, generally capable navy.<br> The Terran Government traces back to the Intercelestial Space Authority, an organization created to legislate \
+    and enforce Space Law, which over time amassed more influence until eventually managing to absorb most of the UN, leading to the modern-day government. \
+    The Terran Government is the largest human authority in space, currently fighting a war against the ICC, one that has however been put on delay due to \
+    other threats. The TG has suffered several losses at the hands of the ''xenos'', and it remains to be seen whether they can take them and the ICC on or not.<br> \
+    mechanics_text = "11 billion people.<br><br>Standing army of several million troops.<br><br>~600 FTL ships, mainly modern and combat-ready<br><br>\
+    ++Largest military in space.<br>\
+    +Good scientific capabilities.<br>\
+    +Possesses GalComs, allowing FTL communication.<br>\
+    -High opposition and unrest in fringe areas.<br>\
+    --Navy is spread out thinly."
+    
+/datum/codex_entry/NT_faction
+    display_name = "NanoTrasen (faction)"
+    lore_text = "NanoTrasen, known informally as NT, is the largest corporation in human space, with facilities mainly in TG space along with a few \
+    frontier outposts. They focus on phoron research and mining.<br> NanoTrasen was started in in 2057, their main invention back then being a phoron-engine design. \
+    Ever since, NT has gained the favor of the ISA and TG, allowing them to rapidly expand operations, eventually encompasssing most human space. \
+    Despite their generally good PR and wide acclaim, vocal critics of NT indicate their abuses of workers, as well as their controversial production of \
+    vatborns and synthetics for the military. Despite this, NT is projected to keep growing, even with the TG-ICC war.<br> \
```

Comment:
>Despite their generally good PR and wide acclaim, vocal critics of NT indicate their abuses of workers, as well as their controversial production of \ vatborns and synthetics for the military.

Synths aren't made by NT. As of right now in the timeline, NT isn't the super mega-corp that it is in "present day" SS13. Ideally synths are produced by a different manufacturer and NT is still just in charge of interstellar travel and research. Like the SpaceX of SS13